pkg/otlp/config.go

```diff
@@ -163,7 +164,15 @@ func fromExperimentalConfig(cfg config.Config) (PipelineConfig, error) {
 // IsEnabled checks if OTLP pipeline is enabled in a given config.
 func IsEnabled(cfg config.Config) bool {
 	// TODO (AP-1267): Check stable config too
-	return isSetExperimental(cfg)
+	isSet := isSetExperimental(cfg)
+	// NOTE: it is perhaps not ideal to set this here, but:
+	// * we can't import otlp in inventories, because that triggers import loop
```

Comment:
😿
What about some kind of register-a-callback mechanism?

Reply:
Yeah, that would definitely work. Since this is only the first case in which we see the need to do so, I'd prefer to solve it the way Albert suggested above, but if we see more cases like this, we probably should create a callback mechanism.